recipes/rang/all/conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile, Meson, tools
+import os
+
+
+class RangConan(ConanFile):
+    name = "rang"
+    description = "A Minimal, Header only Modern c++ library for colors in your terminal"
+    homepage = "https://github.com/agauniyal/rang"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
```

Comment:
```suggestion
    license = "Unlicense"
```

https://github.com/agauniyal/rang/blob/master/LICENSE
https://spdx.org/licenses/Unlicense.html